diff --git a/node_modules/@embroider/core/src/html-entrypoint.js b/node_modules/@embroider/core/src/html-entrypoint.js
index 8dd5776..ecae0f2 100644
--- a/node_modules/@embroider/core/src/html-entrypoint.js
+++ b/node_modules/@embroider/core/src/html-entrypoint.js
@@ -48,7 +48,7 @@ class HTMLEntrypoint {
         }
     }
     relativeToApp(rootRelativeURL) {
-        return rootRelativeURL.replace(this.rootURL, '');
+      return rootRelativeURL.replace(new RegExp(`^${this.rootURL}`), './');
     }
     handledScripts() {
         let scriptTags = [...this.dom.window.document.querySelectorAll('script')];
