diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index 2d4f2118bf0a27b63315b7788ffd7a8b2728ffb0..4fff9fa7fb85b00f141ad77dc7682916942b5dc4 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -105,9 +105,6 @@ function solidPlugin(options = {}) {
           exclude: solidPkgsConfig.optimizeDeps.exclude
         },
         ssr: solidPkgsConfig.ssr,
-        ...(test.server ? {
-          test
-        } : {})
       };
     },
     // @ts-ignore This hook only works in Vite 6
